### PR TITLE
ERPimage and Toposeries

### PR DESCRIPTION
**Pull request by vladdez**
- Interactivity of toposeries was fixed
- Refactorig of ERP image code
- error managment for NaN sorting values
- #200 
- #192 
- configs for additional plots are added

### CODE REVIEW

File: src/eeg_series.jl [hunk 1]
```diff
@@ -219,9 +221,10 @@ function scatter_management(
                 merge(topoplot_attributes, (; label_scatter = highlight_feature))
         end
     end
+    return topoplot_attributes
```
Comment (behinger) [inline src/eeg_series.jl:224]: why return them?

File: src/plot_erpimage.jl [hunk 2]
```diff
@@ -112,78 +104,109 @@ function plot_erpimage!(
...
+function ei_meanplot(ax, data, config, f, ga, times, show_sortval)
+    ax.xlabelvisible = false #padding of the main plot
+    ax.xticklabelsvisible = false
+
+    trace = @lift(mean($data, dims = 2)[:, 1])
+    axbottom = Axis(
+        ga[5, 1:4];
+        height = 100,
+        ylabel = config.colorbar.label === nothing ? "" : config.colorbar.label,
+        xlabel = "Time [s]",
```
Comment (behinger) [inline src/plot_erpimage.jl:138]: there are quite some hardcoded things. Maybe move them to PlotConfig similar to the inset-options for the topoplot in butterflyplot?

**Review by behinger — APPROVED** [review]

**Comment by behinger** [comment]
I think topoplotseries needs some high-level documentation on how it is implemented, I still cannot really understand the flow of information - but maybe for another PR

**Comment by behinger** [comment]
Tests are failling

File: src/eeg_series.jl (hunk 1 above)
Comment (vladdez) [reply src/eeg_series.jl:224]: it will crash otherwise

**Pull request merged**